make/autoconf/basic.m4

```diff
@@ -479,7 +479,7 @@ AC_DEFUN([BASIC_CHECK_DIR_ON_LOCAL_DISK],
     # df on AIX does not understand -l. On modern AIXes it understands "-T local" which
     # is the same. On older AIXes we just continue to live with a "not local build" warning.
     if test "x$OPENJDK_TARGET_OS" = xaix; then
-      if "$DF -T local > /dev/null 2>&1"; then
+      if ""$DF" -T local > /dev/null 2>&1"; then
```

Comment:
```suggestion
      if $DF -T local > /dev/null 2>&1; then
```
I can't test this, but I don't think the quotes serve any purpose here.

Reply:
@erikj79 Thanks for reviewing. I verified this test and it worked fine.